app/jobs/delete_missed_reservations_job.rb

```diff
@@ -0,0 +1,31 @@
+class DeleteMissedReservationsJob < ReservationJob
+  private
+
+  def enabled
+    !AppConfig.check(:res_exp_time, '').blank?
+  end
+
+  def log_start
+    old_missed_reservations
```

Comment:
It's a little confusing to me that we're calling this so that the collection is memoized so we can count it in the log message; not confusing in the sense that I don't understand why that happens, but confusing in the sense that it wasn't clear why this belonged in the `log_start` method. Should we add an additional step called `prep_collection` or something in `ReservationJob` so that it's a bit clearer? It might just be a matter of rearranging some things, since we're already basically doing it, or maybe just using method aliases so we still get the readability in `run` of operating on `old_missed_reservations` without also calling that method here.